lightning/src/ln/channel.rs

```diff
@@ -301,6 +301,21 @@ pub struct CounterpartyForwardingInfo {
 	pub cltv_expiry_delta: u16,
 }
 
+/// When a node increases the feerate on an outbound channel, it can cause the channel to get
```

Comment:
nit: "Updating fee and paying for them is the responsibility is on the channel initiator in the `update_fee` fee-bumping schemes. When it does increase the feerate  on an outbound channel", good to quickly recall on who is the fee-burden

Reply:
I rewrote it a chunk.